dask/dataframe/core.py

```diff
@@ -1538,14 +1538,35 @@ def sample(self, n=None, frac=None, replace=False, random_state=None):
         n : int, optional
             Number of items to return is not supported by dask. Use frac
             instead.
-        frac : float, optional
+        frac : frac : float, optional
             Fraction of axis items to return.
+            .. warning::
+            Since `sample` is mapped to each partition, `frac` should be greater
+            or equal to the number of partitions divided by the length of the
+            DataFrame or Series object (see below).
         replace : boolean, optional
```

Comment:
```suggestion
            Fraction of axis items to return.
            
            .. warning::
            
               Since `sample` is mapped to each partition, `frac` should be greater
               or equal to the number of partitions divided by the length of the
               DataFrame or Series object (see below).
            
        replace : boolean, optional
```